lib/feedjira/parser/atom.rb

```diff
@@ -14,7 +14,7 @@ class Atom
       elements :entry, as: :entries, class: AtomEntry
 
       def self.able_to_parse?(xml)
-        %r{\<feed[^\>]+xmlns\s?=\s?[\"\'](http://www\.w3\.org/2005/Atom|http://purl\.org/atom/ns\#)[\"\'][^\>]*\>} =~ xml # rubocop:disable Metrics/LineLength
+        %r{\<(?:\S*\:)?feed[^\>]+xmlns(?:\:\S*)?\s?=\s?[\"\'](http://www\.w3\.org/2005/Atom|http://purl\.org/atom/ns\#)[\"\'][^\>]*\>} =~ xml # rubocop:disable Metrics/LineLength
```

Comment:
Oh man. This somewhat cryptic regex is getting more complicated. I wonder if there is a way to achieve this and improve the readability here.